common/app/common/configuration.scala

```diff
@@ -244,11 +244,6 @@ class GuardianConfiguration extends Logging {
     lazy val subscribeWithGoogleApiUrl = configuration.getStringProperty("google.subscribeWithGoogleApiUrl").getOrElse("https://swg.theguardian.com")
   }
 
-  object oriel {
-    lazy val orielApiKey = configuration.getStringProperty("oriel.api.key")
```

Comment:
Will need to remove this from the parameter store.

Reply:
Can you point me to it?

Reply:
I can try ...